scripts/run_integration_tests.sh

```diff
@@ -13,7 +13,7 @@ function cleanup () {
 
 RUNDIR=$(dirname ${0})
 SKIP_JS_BUILD=
-trap cleanup EXIT KILL INT
+trap cleanup KILL INT
```

Comment:
TODO: check this after refreshing merge from main. There is a pending PR from @mattheworris that will probably update this & be merged before this one.